<html xmlns="http://www.w3.org/1999/xhtml"><head>
        <meta http-equiv="Content-Type" content="text/html; charset=utf-8">
        <meta name="viewport" content="width=device-width, height=device-height, initial-scale=0.5, minimum-scale=0.5, user-scalable=yes, maximum-scale=2.0, target-densitydpi=device-dpi" />
				<meta name="robots" content="all"/>
        <meta name="description" content="description">
        <meta name="keywords" content="">
        <title>Page 49 - Thesis_JEVI</title>
<style type="text/css">
<!--
body {
	margin-left: 0px;
	margin-top: 0px;
	margin-right: 0px;
	margin-bottom: 0px;
}
  
#container {margin:50px auto;padding:0px;width:910px;}
#left-button{ width:90px; height:100%;float:left }
#middle{ width:730px; margin:0px; float: left;}
#right-button{ width:90px; float:left;}
html, body, div, ul, li, dl, dt, dd, h1, h2, h3, h4, h5, h6, form, input, dd,select, button, textarea, iframe, table, th, td { margin: 0; padding: 0; }
img { border: 0 none; displayblock}
span,ul,li,image{ padding:0px; margin:0px; list-style:none;}
#top-title{ width:714px; margin-bottom:10px; float:left; padding:0px 8px}
#top-title2{ width:320px;  float:right;}
#content{ width: 730px; float:left;float:left;word-wrap: break-word;}
#content-top{ width:730px; background-image:url(images/top_bg.jpg); height:56px; background-repeat:no-repeat; float:left;}
#content-top1{ width:70%; padding:16px 0px 0px 30px; font-family:Arial, Helvetica, sans-serif; font-size:24px; color:#636363;float:left;}
#content-top2{ width:60px; float:right; padding:25px 30px 0px 0px;font-family:Arial, Helvetica, sans-serif; font-size:11px; color:#2f2f2f; font-weight:bold; text-align:right}
#content-middle{ width:730px; background-image:url(images/middle_bg.jpg); background-repeat:repeat-y;float:left;}
#content-middle1{ width:475px; padding:25px 0px 25px 35px; font-family:Arial, Helvetica, sans-serif; font-size:12px; color:#636363; line-height:150%;float:left;}
#content-middle2{  padding:7px; float:right; margin:25px 35px 0px 0px; border:solid 1px #c8c8c9; background-color:#FFF; text-align:center}
#content-bottom{ width:730px; background-image:url(images/bottom_bg.jpg); background-repeat:no-repeat; height:58px;float:left;}
#content-bottom2{ width:110px; height:42px; margin:0px auto; padding-top:4px;}
#input{ width:58px; height:20px; background-color:#FFF; border:solid 1px #dfdfe0; color:#636363; text-align:center; font-size:11px;}
.font1{ font-family:Arial, Helvetica, sans-serif; font-size:10px; color:#2f2f2f;}
.font1 a:link{
	text-decoration: none;
    color:#2f2f2f;

}
.font1 a:visited{
	text-decoration: none;
    color:#2f2f2f;
	
}
.font1 a:hover{
	color: #2f2f2f;
	text-decoration: underline;

}



.text-container {
padding: 20px;
color: #333;
margin-top: 24px;
line-height: 22px;
border-top-left-radius: 3px;
border-top-right-radius: 3px;
}

.use-as-video {
float: right;
padding-left: 20px;
}
.cc-float-fix:before, .cc-float-fix:after {
content: "";
line-height: 0;
height: 0;
width: 0;
display: block;
visibility: hidden;
}
.use-as-video-img {
cursor: pointer;
position: relative;
width: 320px;
height: 180px;
}
.use-as-video-title {
font-size: 15px;
font-weight: bold;
color: #000000;
padding-bottom: 5px;
}
-->
</style>
<script type="text/javascript">
<!--
function MM_preloadImages() { //v3.0
  var d=document; if(d.images){ if(!d.MM_p) d.MM_p=new Array();
    var i,j=d.MM_p.length,a=MM_preloadImages.arguments; for(i=0; i<a.length; i++)
    if (a[i].indexOf("#")!=0){ d.MM_p[j]=new Image; d.MM_p[j++].src=a[i];}}
}

function MM_swapImgRestore() { //v3.0
  var i,x,a=document.MM_sr; for(i=0;a&&i<a.length&&(x=a[i])&&x.oSrc;i++) x.src=x.oSrc;
}

function MM_findObj(n, d) { //v4.01
  var p,i,x;  if(!d) d=document; if((p=n.indexOf("?"))>0&&parent.frames.length) {
    d=parent.frames[n.substring(p+1)].document; n=n.substring(0,p);}
  if(!(x=d[n])&&d.all) x=d.all[n]; for (i=0;!x&&i<d.forms.length;i++) x=d.forms[i][n];
  for(i=0;!x&&d.layers&&i<d.layers.length;i++) x=MM_findObj(n,d.layers[i].document);
  if(!x && d.getElementById) x=d.getElementById(n); return x;
}

function MM_swapImage() { //v3.0
  var i,j=0,x,a=MM_swapImage.arguments; document.MM_sr=new Array; for(i=0;i<(a.length-2);i+=3)
   if ((x=MM_findObj(a[i]))!=null){document.MM_sr[j++]=x; if(!x.oSrc) x.oSrc=x.src; x.src=a[i+2];}
}
//-->
</script>
</head>

<body onload="MM_preloadImages('images/next2.png','images/previous2.png')">
<div id="container">
<div id="left-button">
  <p>&nbsp;</p>
  <p>&nbsp;</p>
  <p>&nbsp;</p>
  <p>&nbsp;</p>
  <p>&nbsp;</p>
  <p><a href="page48.html"><img src="images/previous_big.png" width="90" height="80" /></a></p>
</div>
<div id="middle">
  <div id="top-title"><span class="font1">Basic HTML Version</span>
    <div id="top-title2">
      <table width="100%" border="0" align="right" cellpadding="0" cellspacing="0">
        <tr>
        	<td width="66%"> </td>
          <td width="35%" style="display:none" align="right"><img src="images/content.png" width="19" height="16" /></td>
          <td width="31%" style="display:none" class="font1"><a href="">Table of Contents</a></td>
          <td width="6%"><img src="images/view.png" width="19" height="16" /></td>
          <td width="28%" class="font1"><a href="../../Thesis_JEVI.html#p=49">View Full Version</a></td>
        </tr>
      </table>
    </div>
  </div>
  <div id="content">
     <div id="content-top">
        <div id="content-top1">Page 49 - Thesis_JEVI</div>
        <div id="content-top2">P. 49</div>
     </div>
     <div id="content-middle">
     	
     	  <div class="text-container">
    <div class="use-as-video cc-float-fix">
      <a style="text-decoration: none;" href="../../Thesis_JEVI.html#p=49"><img src="../thumb/49.jpg" /></a>
    </div>

    <p class="use-as-video-title"></p>
    <pre style="white-space:pre-line"><code>AN ORGANOID PLATFORM FOR OVARIAN CANCER CAPTURES INTRA- AND INTERPATIENT HETEROGENEITY



               SNVs that were only detected in either the tumor or the organoid sample of a pair were
               called in the corresponding sample (tumor or organoid) when supported by more than
               5% of the reads covering that position. CNV was detected for each sample independently
                                      261
               using Control-FREEC, v. 7.2  and assuming a ploidy of 2. For sample HGS-6, a ploidy
               of 3 was assumed for the plots. Structural variation calling was performed using Manta,
 2             v.0.29.5 . For increased sensitivity, we ran Manta in the four available analysis types:   2
                     262
               singlesample, multi-sample, tumor-only and tumor-normal. When comparing SVs
               called in one of the tumor/organoid pairs with the matching sample, we inspected the
               output of the tumor-normal mode of the pertinent tumor/organoid sample with the re-
               sults of the four calling modes for the matching tumor/organoid sample. Somatic variant
               calling could not be performed for samples without matching reference DNA (CCC-1
               and END-1). In these cases, germline variant calling was performed jointly for tumor
                                                                   255
               and organoid samples using GATK’s Haplotype Caller, v3.4-46 . Germline calls were
                                                                                 264
                                                            263
               filtered against the Genome of the Netherlands (GoNL)  and the 1000 Genomes  and
               only variants with a predicted ‘moderate’ or ‘high’ effect (SnpEff v.4.1 ) were kept. For
                                                                       260
               SV calling of the CCC-1 and END-1 samples, the tumor-normal mode of Manta could
               not be used, but all other Manta variant calling workflows were performed (tumor-only,
               singlesample, multi-sample). To enrich for somatic SVs, only SVs larger than 10 Kb and
               not found in the GoNL or 1000 Genomes studies were considered for these two samples.


               Single-cell WGS library preparation


               Cells were sorted into 384-well plates with 5 μl of mineral oil (Sigma-Aldrich). After
               sorting cells, can be stored at −20 °C. Five-hundred nanoliters of lysis mix (0.001 U μl
                                                                                     –1
               Qiagen Protease in NEB Buffer 4) was added to each well and lysis was performed at 55
               °C overnight followed by heat inactivation for 20 min at 75 °C and for 5 min at 80 °C.
               Five-hundred nanoliters of Restriction Enzyme mix (1 U μl–1 NLAIII in NEB Cutsmart
               buffer) was added to each well and restriction was performed for 3 h at 37 °C followed by
               heat inactivation for 20 min at 65 °C. One-hundred nanoliters of 1 μM barcoded double-
               stranded NLAIII adapter was added to each well. Ligation mix (1,100 μl, 182 U μl  T4
                                                                                  –1
               DNA Ligase in 1x T4 DNA Ligase buffer supplemented with 3 mM ATP) was added to
               each well and ligation was performed overnight at 16 °C. After ligation, single cells were
               pooled and library preparation was performed as described in Muraro et al. . Libraries
                                                                            265
               were sequenced on an Illumina Nextseq500 with 2 x 75-bp paired-end sequencing.

               Single-cell WGS data analysis


               Reads were aligned to GRCh38 using Burrows– Wheeler Aligner v0.7.14 mapping tool
               with settings ‘bwa mem –M’ . Data were binned in 1 MB bins and normalized to the
                                      266

                                                                                       41</code></pre>
     	</div>
    
     </div>
     <div id="content-bottom">
       <div id="content-bottom2">
         <table width="110" border="0" cellspacing="0" cellpadding="0">
           <tr>
             <td width="22%" height="44"><a href="page48.html"><img src="images/previous.png" width="26" height="19" id="Image2" onmouseover="MM_swapImage('Image2','','images/previous2.png',1)" onmouseout="MM_swapImgRestore()" /></a></td>
             <td width="43%">
             	&nbsp;&nbsp;&nbsp;<a href="page44.html">44</a>&nbsp;&nbsp;&nbsp;<a href="page45.html">45</a>&nbsp;&nbsp;&nbsp;<a href="page46.html">46</a>&nbsp;&nbsp;&nbsp;<a href="page47.html">47</a>&nbsp;&nbsp;&nbsp;<a href="page48.html">48</a>&nbsp;&nbsp;&nbsp;<a href="page49.html">49</a>&nbsp;&nbsp;&nbsp;<a href="page50.html">50</a>&nbsp;&nbsp;&nbsp;<a href="page51.html">51</a>&nbsp;&nbsp;&nbsp;<a href="page52.html">52</a>&nbsp;&nbsp;&nbsp;<a href="page53.html">53</a>&nbsp;&nbsp;&nbsp;<a href="page54.html">54</a>
             </td>
             <td width="35%"><a href="page50.html"><img src="images/next.png" width="26" height="19" id="Image1" onmouseover="MM_swapImage('Image1','','images/next2.png',1)" onmouseout="MM_swapImgRestore()" /></a></td>
           </tr>
         </table>
       </div>
     </div>
  </div>
</div>
<div id="right-button">
  <p>&nbsp;</p>
  <p>&nbsp;</p>
  <p>&nbsp;</p>
  <p>&nbsp;</p>
  <p>&nbsp;</p>
  <p><a href="page50.html"><img src="images/next_big.png" width="90" height="80" /></a></p>
</div>


</div>

</body>
</html>
