<html xmlns="http://www.w3.org/1999/xhtml"><head>
        <meta http-equiv="Content-Type" content="text/html; charset=utf-8">
        <meta name="viewport" content="width=device-width, height=device-height, initial-scale=0.5, minimum-scale=0.5, user-scalable=yes, maximum-scale=2.0, target-densitydpi=device-dpi" />
				<meta name="robots" content="all"/>
        <meta name="description" content="description">
        <meta name="keywords" content="">
        <title>Page 38 - Thesis_JEVI</title>
<style type="text/css">
<!--
body {
	margin-left: 0px;
	margin-top: 0px;
	margin-right: 0px;
	margin-bottom: 0px;
}
  
#container {margin:50px auto;padding:0px;width:910px;}
#left-button{ width:90px; height:100%;float:left }
#middle{ width:730px; margin:0px; float: left;}
#right-button{ width:90px; float:left;}
html, body, div, ul, li, dl, dt, dd, h1, h2, h3, h4, h5, h6, form, input, dd,select, button, textarea, iframe, table, th, td { margin: 0; padding: 0; }
img { border: 0 none; displayblock}
span,ul,li,image{ padding:0px; margin:0px; list-style:none;}
#top-title{ width:714px; margin-bottom:10px; float:left; padding:0px 8px}
#top-title2{ width:320px;  float:right;}
#content{ width: 730px; float:left;float:left;word-wrap: break-word;}
#content-top{ width:730px; background-image:url(images/top_bg.jpg); height:56px; background-repeat:no-repeat; float:left;}
#content-top1{ width:70%; padding:16px 0px 0px 30px; font-family:Arial, Helvetica, sans-serif; font-size:24px; color:#636363;float:left;}
#content-top2{ width:60px; float:right; padding:25px 30px 0px 0px;font-family:Arial, Helvetica, sans-serif; font-size:11px; color:#2f2f2f; font-weight:bold; text-align:right}
#content-middle{ width:730px; background-image:url(images/middle_bg.jpg); background-repeat:repeat-y;float:left;}
#content-middle1{ width:475px; padding:25px 0px 25px 35px; font-family:Arial, Helvetica, sans-serif; font-size:12px; color:#636363; line-height:150%;float:left;}
#content-middle2{  padding:7px; float:right; margin:25px 35px 0px 0px; border:solid 1px #c8c8c9; background-color:#FFF; text-align:center}
#content-bottom{ width:730px; background-image:url(images/bottom_bg.jpg); background-repeat:no-repeat; height:58px;float:left;}
#content-bottom2{ width:110px; height:42px; margin:0px auto; padding-top:4px;}
#input{ width:58px; height:20px; background-color:#FFF; border:solid 1px #dfdfe0; color:#636363; text-align:center; font-size:11px;}
.font1{ font-family:Arial, Helvetica, sans-serif; font-size:10px; color:#2f2f2f;}
.font1 a:link{
	text-decoration: none;
    color:#2f2f2f;

}
.font1 a:visited{
	text-decoration: none;
    color:#2f2f2f;
	
}
.font1 a:hover{
	color: #2f2f2f;
	text-decoration: underline;

}



.text-container {
padding: 20px;
color: #333;
margin-top: 24px;
line-height: 22px;
border-top-left-radius: 3px;
border-top-right-radius: 3px;
}

.use-as-video {
float: right;
padding-left: 20px;
}
.cc-float-fix:before, .cc-float-fix:after {
content: "";
line-height: 0;
height: 0;
width: 0;
display: block;
visibility: hidden;
}
.use-as-video-img {
cursor: pointer;
position: relative;
width: 320px;
height: 180px;
}
.use-as-video-title {
font-size: 15px;
font-weight: bold;
color: #000000;
padding-bottom: 5px;
}
-->
</style>
<script type="text/javascript">
<!--
function MM_preloadImages() { //v3.0
  var d=document; if(d.images){ if(!d.MM_p) d.MM_p=new Array();
    var i,j=d.MM_p.length,a=MM_preloadImages.arguments; for(i=0; i<a.length; i++)
    if (a[i].indexOf("#")!=0){ d.MM_p[j]=new Image; d.MM_p[j++].src=a[i];}}
}

function MM_swapImgRestore() { //v3.0
  var i,x,a=document.MM_sr; for(i=0;a&&i<a.length&&(x=a[i])&&x.oSrc;i++) x.src=x.oSrc;
}

function MM_findObj(n, d) { //v4.01
  var p,i,x;  if(!d) d=document; if((p=n.indexOf("?"))>0&&parent.frames.length) {
    d=parent.frames[n.substring(p+1)].document; n=n.substring(0,p);}
  if(!(x=d[n])&&d.all) x=d.all[n]; for (i=0;!x&&i<d.forms.length;i++) x=d.forms[i][n];
  for(i=0;!x&&d.layers&&i<d.layers.length;i++) x=MM_findObj(n,d.layers[i].document);
  if(!x && d.getElementById) x=d.getElementById(n); return x;
}

function MM_swapImage() { //v3.0
  var i,j=0,x,a=MM_swapImage.arguments; document.MM_sr=new Array; for(i=0;i<(a.length-2);i+=3)
   if ((x=MM_findObj(a[i]))!=null){document.MM_sr[j++]=x; if(!x.oSrc) x.oSrc=x.src; x.src=a[i+2];}
}
//-->
</script>
</head>

<body onload="MM_preloadImages('images/next2.png','images/previous2.png')">
<div id="container">
<div id="left-button">
  <p>&nbsp;</p>
  <p>&nbsp;</p>
  <p>&nbsp;</p>
  <p>&nbsp;</p>
  <p>&nbsp;</p>
  <p><a href="page37.html"><img src="images/previous_big.png" width="90" height="80" /></a></p>
</div>
<div id="middle">
  <div id="top-title"><span class="font1">Basic HTML Version</span>
    <div id="top-title2">
      <table width="100%" border="0" align="right" cellpadding="0" cellspacing="0">
        <tr>
        	<td width="66%"> </td>
          <td width="35%" style="display:none" align="right"><img src="images/content.png" width="19" height="16" /></td>
          <td width="31%" style="display:none" class="font1"><a href="">Table of Contents</a></td>
          <td width="6%"><img src="images/view.png" width="19" height="16" /></td>
          <td width="28%" class="font1"><a href="../../Thesis_JEVI.html#p=38">View Full Version</a></td>
        </tr>
      </table>
    </div>
  </div>
  <div id="content">
     <div id="content-top">
        <div id="content-top1">Page 38 - Thesis_JEVI</div>
        <div id="content-top2">P. 38</div>
     </div>
     <div id="content-middle">
     	
     	  <div class="text-container">
    <div class="use-as-video cc-float-fix">
      <a style="text-decoration: none;" href="../../Thesis_JEVI.html#p=38"><img src="../thumb/38.jpg" /></a>
    </div>

    <p class="use-as-video-title"></p>
    <pre style="white-space:pre-line"><code>CHAPTER 2



            plot of single-cell CNV profiles from two recurrent tumor samples and corresponding organoid lines (HGS-
            1-R2, HGS-1-R3) of a single patient. Hierarchical clustering has separated the cells into five different clusters
            (color coded). Total number of analyzed cells is 791. (C) t-SNE plot presenting diploid (green) and aneuploid
            (red) cells. Total number of analyzed cells is 791. (D) Single-cell distribution into the different clusters
            according to sample of origin. T, tumor; O, organoid; -s, second time point analysis. HGS-1-R2_T, n = 351
            cells; HGS-1-R2_O, passage 5, n = 159 cells; HGS-1-R3_T, n = 93 cells; HGS-1-R3_O, passage 4, n = 122
            cells; HGS-1-R3_O, passage 12, n = 66 cells. (E) Clustered CNV heat map of aneuploid cells presenting gains
     2      (blue) and losses (red) across the genome. Sample origin and cluster belonging of each cell is color coded.                                                                     2

            Clusters 1–4 comprised aneuploid cells whereas cluster 5 comprised dip-
            loid cells (Figure  4c). As expected, tumor samples that were obtained from as-
            cites drainage of a single patient within a one-month interval overlapped with
            each other and did not form separate clusters (Figure 4d), thus validating the
            robustness of the single-cell DNA sequencing method. Organoid-derived cells over-
            lapped with the same five clusters (albeit with low representation in cluster 3) demon-
            strating both their heterogeneity and resemblance to the original tumor samples (Fig-
            ure 4d). HGS-1-R3 relative cell abundance in cluster 5 (diploid cells) was dramatically
            reduced after extended passaging (passage four versus 12), whereas representation of
            clusters 1, 2 and 4 (aneuploid cells) increased (Figure 4d,e), suggesting that tumor cells
            overgrew normal cells over time, while maintaining tumor heterogeneity.



            Gene expression analysis of OC organoids


            To assess organoid gene expression profiles, we performed RNA sequencing (RNA-seq)
            on 35 OC organoids, and six normal OSE and FT organoids. Hierarchical clustering as-
            signed organoids to three independent main groups, representing (1) HGS carcinomas,
            (2) MC and END tumors and (3) mainly LGS carcinomas, FT and OSE (Figure 5). Or-
            ganoids derived from multiple tumor lesions of the same patient were transcriptionally
            more similar to each other than to unrelated organoid lines (for example, MC-1.1,2 and
            HGS-3.1,2). In a similar manner, organoids that were sequenced at a second time point
            after extended passaging clustered with their corresponding samples (HGS-1, passage
            eight versus 32; HGS-3.1, passage four versus 32; HGS-1-R2 passage four versus 17). Of
            note, non-malignant MBT and malignant MC organoids clustered together. This was
            seen in eight organoid lines derived from four different patients (two MC and two MBT),
            suggesting a biological link between these samples. This finding is in agreement with a
            causality hypothesis that suggests a stepwise progression from BTs to invasive carcino-
            mas 239–241 . Furthermore, OSE(P)7 organoids (derived from a sample collected during
            risk-reducing salpingo-oophorectomy) clustered together with OC organoids and apart
            from normal OSE and FT organoid lines. This finding, together with morphological,
            histological and metaphase spread analysis (Extended Data Figure 3e,f), suggested that
            OSE(P)7 consists of malignant cells that were not diagnosed by routine pathological ex-


          30</code></pre>
     	</div>
    
     </div>
     <div id="content-bottom">
       <div id="content-bottom2">
         <table width="110" border="0" cellspacing="0" cellpadding="0">
           <tr>
             <td width="22%" height="44"><a href="page37.html"><img src="images/previous.png" width="26" height="19" id="Image2" onmouseover="MM_swapImage('Image2','','images/previous2.png',1)" onmouseout="MM_swapImgRestore()" /></a></td>
             <td width="43%">
             	&nbsp;&nbsp;&nbsp;<a href="page33.html">33</a>&nbsp;&nbsp;&nbsp;<a href="page34.html">34</a>&nbsp;&nbsp;&nbsp;<a href="page35.html">35</a>&nbsp;&nbsp;&nbsp;<a href="page36.html">36</a>&nbsp;&nbsp;&nbsp;<a href="page37.html">37</a>&nbsp;&nbsp;&nbsp;<a href="page38.html">38</a>&nbsp;&nbsp;&nbsp;<a href="page39.html">39</a>&nbsp;&nbsp;&nbsp;<a href="page40.html">40</a>&nbsp;&nbsp;&nbsp;<a href="page41.html">41</a>&nbsp;&nbsp;&nbsp;<a href="page42.html">42</a>&nbsp;&nbsp;&nbsp;<a href="page43.html">43</a>
             </td>
             <td width="35%"><a href="page39.html"><img src="images/next.png" width="26" height="19" id="Image1" onmouseover="MM_swapImage('Image1','','images/next2.png',1)" onmouseout="MM_swapImgRestore()" /></a></td>
           </tr>
         </table>
       </div>
     </div>
  </div>
</div>
<div id="right-button">
  <p>&nbsp;</p>
  <p>&nbsp;</p>
  <p>&nbsp;</p>
  <p>&nbsp;</p>
  <p>&nbsp;</p>
  <p><a href="page39.html"><img src="images/next_big.png" width="90" height="80" /></a></p>
</div>


</div>

</body>
</html>
