<html xmlns="http://www.w3.org/1999/xhtml"><head>
        <meta http-equiv="Content-Type" content="text/html; charset=utf-8">
        <meta name="viewport" content="width=device-width, height=device-height, initial-scale=0.5, minimum-scale=0.5, user-scalable=yes, maximum-scale=2.0, target-densitydpi=device-dpi" />
				<meta name="robots" content="all"/>
        <meta name="description" content="description">
        <meta name="keywords" content="">
        <title>Page 162 - Thesis_JEVI</title>
<style type="text/css">
<!--
body {
	margin-left: 0px;
	margin-top: 0px;
	margin-right: 0px;
	margin-bottom: 0px;
}
  
#container {margin:50px auto;padding:0px;width:910px;}
#left-button{ width:90px; height:100%;float:left }
#middle{ width:730px; margin:0px; float: left;}
#right-button{ width:90px; float:left;}
html, body, div, ul, li, dl, dt, dd, h1, h2, h3, h4, h5, h6, form, input, dd,select, button, textarea, iframe, table, th, td { margin: 0; padding: 0; }
img { border: 0 none; displayblock}
span,ul,li,image{ padding:0px; margin:0px; list-style:none;}
#top-title{ width:714px; margin-bottom:10px; float:left; padding:0px 8px}
#top-title2{ width:320px;  float:right;}
#content{ width: 730px; float:left;float:left;word-wrap: break-word;}
#content-top{ width:730px; background-image:url(images/top_bg.jpg); height:56px; background-repeat:no-repeat; float:left;}
#content-top1{ width:70%; padding:16px 0px 0px 30px; font-family:Arial, Helvetica, sans-serif; font-size:24px; color:#636363;float:left;}
#content-top2{ width:60px; float:right; padding:25px 30px 0px 0px;font-family:Arial, Helvetica, sans-serif; font-size:11px; color:#2f2f2f; font-weight:bold; text-align:right}
#content-middle{ width:730px; background-image:url(images/middle_bg.jpg); background-repeat:repeat-y;float:left;}
#content-middle1{ width:475px; padding:25px 0px 25px 35px; font-family:Arial, Helvetica, sans-serif; font-size:12px; color:#636363; line-height:150%;float:left;}
#content-middle2{  padding:7px; float:right; margin:25px 35px 0px 0px; border:solid 1px #c8c8c9; background-color:#FFF; text-align:center}
#content-bottom{ width:730px; background-image:url(images/bottom_bg.jpg); background-repeat:no-repeat; height:58px;float:left;}
#content-bottom2{ width:110px; height:42px; margin:0px auto; padding-top:4px;}
#input{ width:58px; height:20px; background-color:#FFF; border:solid 1px #dfdfe0; color:#636363; text-align:center; font-size:11px;}
.font1{ font-family:Arial, Helvetica, sans-serif; font-size:10px; color:#2f2f2f;}
.font1 a:link{
	text-decoration: none;
    color:#2f2f2f;

}
.font1 a:visited{
	text-decoration: none;
    color:#2f2f2f;
	
}
.font1 a:hover{
	color: #2f2f2f;
	text-decoration: underline;

}



.text-container {
padding: 20px;
color: #333;
margin-top: 24px;
line-height: 22px;
border-top-left-radius: 3px;
border-top-right-radius: 3px;
}

.use-as-video {
float: right;
padding-left: 20px;
}
.cc-float-fix:before, .cc-float-fix:after {
content: "";
line-height: 0;
height: 0;
width: 0;
display: block;
visibility: hidden;
}
.use-as-video-img {
cursor: pointer;
position: relative;
width: 320px;
height: 180px;
}
.use-as-video-title {
font-size: 15px;
font-weight: bold;
color: #000000;
padding-bottom: 5px;
}
-->
</style>
<script type="text/javascript">
<!--
function MM_preloadImages() { //v3.0
  var d=document; if(d.images){ if(!d.MM_p) d.MM_p=new Array();
    var i,j=d.MM_p.length,a=MM_preloadImages.arguments; for(i=0; i<a.length; i++)
    if (a[i].indexOf("#")!=0){ d.MM_p[j]=new Image; d.MM_p[j++].src=a[i];}}
}

function MM_swapImgRestore() { //v3.0
  var i,x,a=document.MM_sr; for(i=0;a&&i<a.length&&(x=a[i])&&x.oSrc;i++) x.src=x.oSrc;
}

function MM_findObj(n, d) { //v4.01
  var p,i,x;  if(!d) d=document; if((p=n.indexOf("?"))>0&&parent.frames.length) {
    d=parent.frames[n.substring(p+1)].document; n=n.substring(0,p);}
  if(!(x=d[n])&&d.all) x=d.all[n]; for (i=0;!x&&i<d.forms.length;i++) x=d.forms[i][n];
  for(i=0;!x&&d.layers&&i<d.layers.length;i++) x=MM_findObj(n,d.layers[i].document);
  if(!x && d.getElementById) x=d.getElementById(n); return x;
}

function MM_swapImage() { //v3.0
  var i,j=0,x,a=MM_swapImage.arguments; document.MM_sr=new Array; for(i=0;i<(a.length-2);i+=3)
   if ((x=MM_findObj(a[i]))!=null){document.MM_sr[j++]=x; if(!x.oSrc) x.oSrc=x.src; x.src=a[i+2];}
}
//-->
</script>
</head>

<body onload="MM_preloadImages('images/next2.png','images/previous2.png')">
<div id="container">
<div id="left-button">
  <p>&nbsp;</p>
  <p>&nbsp;</p>
  <p>&nbsp;</p>
  <p>&nbsp;</p>
  <p>&nbsp;</p>
  <p><a href="page161.html"><img src="images/previous_big.png" width="90" height="80" /></a></p>
</div>
<div id="middle">
  <div id="top-title"><span class="font1">Basic HTML Version</span>
    <div id="top-title2">
      <table width="100%" border="0" align="right" cellpadding="0" cellspacing="0">
        <tr>
        	<td width="66%"> </td>
          <td width="35%" style="display:none" align="right"><img src="images/content.png" width="19" height="16" /></td>
          <td width="31%" style="display:none" class="font1"><a href="">Table of Contents</a></td>
          <td width="6%"><img src="images/view.png" width="19" height="16" /></td>
          <td width="28%" class="font1"><a href="../../Thesis_JEVI.html#p=162">View Full Version</a></td>
        </tr>
      </table>
    </div>
  </div>
  <div id="content">
     <div id="content-top">
        <div id="content-top1">Page 162 - Thesis_JEVI</div>
        <div id="content-top2">P. 162</div>
     </div>
     <div id="content-middle">
     	
     	  <div class="text-container">
    <div class="use-as-video cc-float-fix">
      <a style="text-decoration: none;" href="../../Thesis_JEVI.html#p=162"><img src="../thumb/162.jpg" /></a>
    </div>

    <p class="use-as-video-title"></p>
    <pre style="white-space:pre-line"><code>CHAPTER 6



            junction. PCR primers are designed on both sides of the fusion. A PCR with breakpoint-spanning
            primers  will only result in a product for the rearrangement.  (B) Schematic outline of
            KMT2A with its two breakpoint clusters (major: red; minor: green). Scissors indicate
            where the crRNAs target Cas9 to cut and arrows indicate the desired sequencing direction.
            (C) Schematic outline of SIL and TAL1. Common breakpoint areas are highlighted by a
            blue box. Scissors indicate where the crRNAs target Cas9 to cut and arrows indicate the
            desired sequencing direction. Genomic regions are not scaled.

            We performed target enrichment for each sample with appropriate sets of crRNAs (Sup-
            plementary Table 1) and sequenced each sample on one Oxford Nanopore Technolo-
            gies MinION flow cell (R9.4). After mapping to the reference genome (GRCh38), the ge-
            nome wide coverage per sample was 0.028x (B-ALL1), 0.04x (B-ALL2), 0.10x (B-ALL3),
            0.54x (B-ALL4), 0.67x (B-ALL5), 0.22x (T-ALL1), 0.17x (T-ALL2), 0.032x (T-ALL3),
            0.12x (T-ALL4) and 0.037x (T-ALL5) (Supplementary Table 1). However, due to the
            targeted sequencing approach, on-target coverages increased to 2-352x for the KMT2A
            locus in the B-ALL samples and 47-290x at the SIL and TAL1 loci in the T-ALL samples.
            (Figure 2A and Supplementary Table 1).
            To detect KMT2A fusion genes we applied NanoFG, which we previously developed to
            identify fusion genes from nanopore sequencing data . NanoFG identified, as expected,
                                                       403
     6      a KMT2A-MLLT1 fusion gene with two fusion-supporting reads in the B-ALL2 sample                                                                                                 6
            (Supplementary Table 2). For the blinded samples B-ALL3 and B-ALL5 we identified
            a KMT2A-MLLT2 fusion with four and 31 fusion-supporting reads respectively, which
            was later confirmed by comparison with outsourced results obtained from a laboratory
            specializing in KMT2A partner detection. In addition to this fusion result, a reciprocal
            MLLT2-KMT2A fusion was detected in the B-ALL5 sample with 28 fusion-supporting
            reads that was not previously detected (Supplementary Figure 2, Supplementary Table
            2). For B-ALL4, the diagnostic laboratory identified a KMT2A-MLLT2 fusion, howev-
            er, neither manual investigation in IGV genome browser  nor NanoFG could confirm
                                                          405
            this fusion within our targeted sequencing data. In accordance with diagnostic efforts,
            NanoFG did not identify genomic evidence of the KMT2A fusion for the B-ALL1 sam-
            ple. We manually confirmed the lack of fusion-spanning reads within the KMT2A and
            MLLT1 genes with IGV. NanoFG is specifically designed to detect fusion genes with
            breakpoints within both of the involved fusion partners.
            Therefore, NanoFG did not call any of the SIL-TAL1 rearrangements within the T-ALL
            samples as the  SIL-TAL1  rearrangement may be induced through breakpoints out-
            side of annotated genes (including promoter, both UTRs, and exonic/intronic regions).
            Therefore, we performed SV calling with Sniffles  within the SIL and TAL1 loci. We
                                                    342
            then identified all five SIL-TAL1 rearrangements with 238x (T-ALL1), 145x (T-ALL2),
            50x (T-ALL3), 177x (T-ALL4) and 49x (T-ALL5) breakpoint-spanning reads (Figure
            2B, Supplementary Table 2). While the breakpoints within SIL were located within the
            5’UTR of the gene, all breakpoints within TAL1 were located about 185 bps upstream


         154</code></pre>
     	</div>
    
     </div>
     <div id="content-bottom">
       <div id="content-bottom2">
         <table width="110" border="0" cellspacing="0" cellpadding="0">
           <tr>
             <td width="22%" height="44"><a href="page161.html"><img src="images/previous.png" width="26" height="19" id="Image2" onmouseover="MM_swapImage('Image2','','images/previous2.png',1)" onmouseout="MM_swapImgRestore()" /></a></td>
             <td width="43%">
             	&nbsp;&nbsp;&nbsp;<a href="page157.html">157</a>&nbsp;&nbsp;&nbsp;<a href="page158.html">158</a>&nbsp;&nbsp;&nbsp;<a href="page159.html">159</a>&nbsp;&nbsp;&nbsp;<a href="page160.html">160</a>&nbsp;&nbsp;&nbsp;<a href="page161.html">161</a>&nbsp;&nbsp;&nbsp;<a href="page162.html">162</a>&nbsp;&nbsp;&nbsp;<a href="page163.html">163</a>&nbsp;&nbsp;&nbsp;<a href="page164.html">164</a>&nbsp;&nbsp;&nbsp;<a href="page165.html">165</a>&nbsp;&nbsp;&nbsp;<a href="page166.html">166</a>&nbsp;&nbsp;&nbsp;<a href="page167.html">167</a>
             </td>
             <td width="35%"><a href="page163.html"><img src="images/next.png" width="26" height="19" id="Image1" onmouseover="MM_swapImage('Image1','','images/next2.png',1)" onmouseout="MM_swapImgRestore()" /></a></td>
           </tr>
         </table>
       </div>
     </div>
  </div>
</div>
<div id="right-button">
  <p>&nbsp;</p>
  <p>&nbsp;</p>
  <p>&nbsp;</p>
  <p>&nbsp;</p>
  <p>&nbsp;</p>
  <p><a href="page163.html"><img src="images/next_big.png" width="90" height="80" /></a></p>
</div>


</div>

</body>
</html>
