<html xmlns="http://www.w3.org/1999/xhtml"><head>
        <meta http-equiv="Content-Type" content="text/html; charset=utf-8">
        <meta name="viewport" content="width=device-width, height=device-height, initial-scale=0.5, minimum-scale=0.5, user-scalable=yes, maximum-scale=2.0, target-densitydpi=device-dpi" />
				<meta name="robots" content="all"/>
        <meta name="description" content="description">
        <meta name="keywords" content="">
        <title>Page 44 - Thesis_JEVI</title>
<style type="text/css">
<!--
body {
	margin-left: 0px;
	margin-top: 0px;
	margin-right: 0px;
	margin-bottom: 0px;
}
  
#container {margin:50px auto;padding:0px;width:910px;}
#left-button{ width:90px; height:100%;float:left }
#middle{ width:730px; margin:0px; float: left;}
#right-button{ width:90px; float:left;}
html, body, div, ul, li, dl, dt, dd, h1, h2, h3, h4, h5, h6, form, input, dd,select, button, textarea, iframe, table, th, td { margin: 0; padding: 0; }
img { border: 0 none; displayblock}
span,ul,li,image{ padding:0px; margin:0px; list-style:none;}
#top-title{ width:714px; margin-bottom:10px; float:left; padding:0px 8px}
#top-title2{ width:320px;  float:right;}
#content{ width: 730px; float:left;float:left;word-wrap: break-word;}
#content-top{ width:730px; background-image:url(images/top_bg.jpg); height:56px; background-repeat:no-repeat; float:left;}
#content-top1{ width:70%; padding:16px 0px 0px 30px; font-family:Arial, Helvetica, sans-serif; font-size:24px; color:#636363;float:left;}
#content-top2{ width:60px; float:right; padding:25px 30px 0px 0px;font-family:Arial, Helvetica, sans-serif; font-size:11px; color:#2f2f2f; font-weight:bold; text-align:right}
#content-middle{ width:730px; background-image:url(images/middle_bg.jpg); background-repeat:repeat-y;float:left;}
#content-middle1{ width:475px; padding:25px 0px 25px 35px; font-family:Arial, Helvetica, sans-serif; font-size:12px; color:#636363; line-height:150%;float:left;}
#content-middle2{  padding:7px; float:right; margin:25px 35px 0px 0px; border:solid 1px #c8c8c9; background-color:#FFF; text-align:center}
#content-bottom{ width:730px; background-image:url(images/bottom_bg.jpg); background-repeat:no-repeat; height:58px;float:left;}
#content-bottom2{ width:110px; height:42px; margin:0px auto; padding-top:4px;}
#input{ width:58px; height:20px; background-color:#FFF; border:solid 1px #dfdfe0; color:#636363; text-align:center; font-size:11px;}
.font1{ font-family:Arial, Helvetica, sans-serif; font-size:10px; color:#2f2f2f;}
.font1 a:link{
	text-decoration: none;
    color:#2f2f2f;

}
.font1 a:visited{
	text-decoration: none;
    color:#2f2f2f;
	
}
.font1 a:hover{
	color: #2f2f2f;
	text-decoration: underline;

}



.text-container {
padding: 20px;
color: #333;
margin-top: 24px;
line-height: 22px;
border-top-left-radius: 3px;
border-top-right-radius: 3px;
}

.use-as-video {
float: right;
padding-left: 20px;
}
.cc-float-fix:before, .cc-float-fix:after {
content: "";
line-height: 0;
height: 0;
width: 0;
display: block;
visibility: hidden;
}
.use-as-video-img {
cursor: pointer;
position: relative;
width: 320px;
height: 180px;
}
.use-as-video-title {
font-size: 15px;
font-weight: bold;
color: #000000;
padding-bottom: 5px;
}
-->
</style>
<script type="text/javascript">
<!--
function MM_preloadImages() { //v3.0
  var d=document; if(d.images){ if(!d.MM_p) d.MM_p=new Array();
    var i,j=d.MM_p.length,a=MM_preloadImages.arguments; for(i=0; i<a.length; i++)
    if (a[i].indexOf("#")!=0){ d.MM_p[j]=new Image; d.MM_p[j++].src=a[i];}}
}

function MM_swapImgRestore() { //v3.0
  var i,x,a=document.MM_sr; for(i=0;a&&i<a.length&&(x=a[i])&&x.oSrc;i++) x.src=x.oSrc;
}

function MM_findObj(n, d) { //v4.01
  var p,i,x;  if(!d) d=document; if((p=n.indexOf("?"))>0&&parent.frames.length) {
    d=parent.frames[n.substring(p+1)].document; n=n.substring(0,p);}
  if(!(x=d[n])&&d.all) x=d.all[n]; for (i=0;!x&&i<d.forms.length;i++) x=d.forms[i][n];
  for(i=0;!x&&d.layers&&i<d.layers.length;i++) x=MM_findObj(n,d.layers[i].document);
  if(!x && d.getElementById) x=d.getElementById(n); return x;
}

function MM_swapImage() { //v3.0
  var i,j=0,x,a=MM_swapImage.arguments; document.MM_sr=new Array; for(i=0;i<(a.length-2);i+=3)
   if ((x=MM_findObj(a[i]))!=null){document.MM_sr[j++]=x; if(!x.oSrc) x.oSrc=x.src; x.src=a[i+2];}
}
//-->
</script>
</head>

<body onload="MM_preloadImages('images/next2.png','images/previous2.png')">
<div id="container">
<div id="left-button">
  <p>&nbsp;</p>
  <p>&nbsp;</p>
  <p>&nbsp;</p>
  <p>&nbsp;</p>
  <p>&nbsp;</p>
  <p><a href="page43.html"><img src="images/previous_big.png" width="90" height="80" /></a></p>
</div>
<div id="middle">
  <div id="top-title"><span class="font1">Basic HTML Version</span>
    <div id="top-title2">
      <table width="100%" border="0" align="right" cellpadding="0" cellspacing="0">
        <tr>
        	<td width="66%"> </td>
          <td width="35%" style="display:none" align="right"><img src="images/content.png" width="19" height="16" /></td>
          <td width="31%" style="display:none" class="font1"><a href="">Table of Contents</a></td>
          <td width="6%"><img src="images/view.png" width="19" height="16" /></td>
          <td width="28%" class="font1"><a href="../../Thesis_JEVI.html#p=44">View Full Version</a></td>
        </tr>
      </table>
    </div>
  </div>
  <div id="content">
     <div id="content-top">
        <div id="content-top1">Page 44 - Thesis_JEVI</div>
        <div id="content-top2">P. 44</div>
     </div>
     <div id="content-middle">
     	
     	  <div class="text-container">
    <div class="use-as-video cc-float-fix">
      <a style="text-decoration: none;" href="../../Thesis_JEVI.html#p=44"><img src="../thumb/44.jpg" /></a>
    </div>

    <p class="use-as-video-title"></p>
    <pre style="white-space:pre-line"><code>CHAPTER 2



            mutated in the TP53 gene, are resistant. Dots represent the mean of technical duplicates. Error bars represent
            s.e.m. of technical duplicates. (D) Heat map of Euclidean distance, based on AUC row Z-score values,
            showing organoid response to a panel of drugs, including PI3K/AKT/mTOR pathway, PARP and Wee1
            inhibitors. n = 18 distinct organoid lines. NT, not tested. (E) Top, dose–response curves for niraparib show
            differential response between organoid lines. Dots represent the mean of technical duplicates. Error bars
            represent s.e.m. of technical duplicates. Bottom, box-and-whisker plot (minimum to maximum) presenting
            RAD51 foci score after radiation. Each point represents percentage of RAD51+ cells within geminin+
     2      (GMN+) cell population in one organoid. Horizontal bars and ‘+’ represent median and mean of all dots,                                                                          2
            respectively. Empty and full dots show results of two biologically independent experiments conducted
            one or two passages apart. Total number (n) of analyzed geminin+ cells in each organoid line is presented.
            (F) Histological analysis of organoid-derived xenograft (HGS-3.1) following orthotopic transplantation
            into the mouse bursa. Tumor cells have invaded into the mouse ovary and H&E staining (left) shows solid
            pattern with indications for slit-like spaces (arrow) as well as pleomorphic cells with prominent nuclear
            atypia. Xenograft has maintained PAX8-positive staining (right). A summary of organoid-derived xenograft
            experiments is presented in Supplementary Table 8. Scale bar, 0.5 mm. (G) Gemcitabine-sensitive organoids
            were subcutaneously injected into immunodeficient mice and tumor size was monitored. Once the tumor
            reached 50 mm3, mice were randomly selected and treated with intraperitoneal injections of gemcitabine (2
            mg per kg body weight) (n = 7 independent mice) or vehicle (n = 9 independent mice), 5 times per week for
            4 consecutive weeks (in total 20 injections). Left, an example of tumor growth over time in a vehicle (white
            dots) and a gemcitabine-treated (black dots) mouse. Right, box-and-whisker plot (minimum to maximum)
            summarizing the results across all vehicle and gemcitabine-treated mice, showing tumor size at day 55.
            Horizontal bars and ‘+’ represent the median and mean of all dots, respectively. *P &lt; 0.001, t-test.


            Discussion


            Developing reliable experimental models that address clinical challenges, such as early
            detection, tumor recurrence and acquired chemotherapy resistance, is a high priority in
            OC research . In this study, we describe an organoid platform that enables long-term in
                       73
            vitro expansion, manipulation and analysis of a wide variety of OC subtypes. A compre-
            hensive analysis demonstrates that OC organoids maintain tumor histological charac-
            teristics, such as nuclear and cellular atypia, and biomarker expression, such as p53 and
            PAX8. Organoids and corresponding tumors remained highly similar at the genomic
            level, even after extended passaging. Furthermore, organoids recapitulated OC hall-
            marks, such as CNVs, recurrent mutations and tumor heterogeneity. Finally, unsuper-
            vised hierarchical clustering of gene expression data grouped the organoids according
            to their tumor type and demonstrated that LGS organoids are more similar to normal
            samples than are HGS lines.


            During organoid biobanking of normal FT and OSE samples, obtained from risk-reduc-
            ing surgeries, we encountered two samples that were apparently malignant: LGS-2 (clin-
            ically diagnosed) and OSE(P)7 (indicated by organoid characterization, Extended Data
            Figure 3e,f). Interestingly, unsupervised hierarchical clustering of gene expression data
            grouped these organoid lines together, thus implying biological similarity. Both organoid
            lines were derived from patients at high risk of developing HGS tumors. Therefore, these
            samples potentially represent an early time point in HGS development. Establishing and


          36</code></pre>
     	</div>
    
     </div>
     <div id="content-bottom">
       <div id="content-bottom2">
         <table width="110" border="0" cellspacing="0" cellpadding="0">
           <tr>
             <td width="22%" height="44"><a href="page43.html"><img src="images/previous.png" width="26" height="19" id="Image2" onmouseover="MM_swapImage('Image2','','images/previous2.png',1)" onmouseout="MM_swapImgRestore()" /></a></td>
             <td width="43%">
             	&nbsp;&nbsp;&nbsp;<a href="page39.html">39</a>&nbsp;&nbsp;&nbsp;<a href="page40.html">40</a>&nbsp;&nbsp;&nbsp;<a href="page41.html">41</a>&nbsp;&nbsp;&nbsp;<a href="page42.html">42</a>&nbsp;&nbsp;&nbsp;<a href="page43.html">43</a>&nbsp;&nbsp;&nbsp;<a href="page44.html">44</a>&nbsp;&nbsp;&nbsp;<a href="page45.html">45</a>&nbsp;&nbsp;&nbsp;<a href="page46.html">46</a>&nbsp;&nbsp;&nbsp;<a href="page47.html">47</a>&nbsp;&nbsp;&nbsp;<a href="page48.html">48</a>&nbsp;&nbsp;&nbsp;<a href="page49.html">49</a>
             </td>
             <td width="35%"><a href="page45.html"><img src="images/next.png" width="26" height="19" id="Image1" onmouseover="MM_swapImage('Image1','','images/next2.png',1)" onmouseout="MM_swapImgRestore()" /></a></td>
           </tr>
         </table>
       </div>
     </div>
  </div>
</div>
<div id="right-button">
  <p>&nbsp;</p>
  <p>&nbsp;</p>
  <p>&nbsp;</p>
  <p>&nbsp;</p>
  <p>&nbsp;</p>
  <p><a href="page45.html"><img src="images/next_big.png" width="90" height="80" /></a></p>
</div>


</div>

</body>
</html>
