<html xmlns="http://www.w3.org/1999/xhtml"><head>
        <meta http-equiv="Content-Type" content="text/html; charset=utf-8">
        <meta name="viewport" content="width=device-width, height=device-height, initial-scale=0.5, minimum-scale=0.5, user-scalable=yes, maximum-scale=2.0, target-densitydpi=device-dpi" />
				<meta name="robots" content="all"/>
        <meta name="description" content="description">
        <meta name="keywords" content="">
        <title>Page 140 - Thesis_JEVI</title>
<style type="text/css">
<!--
body {
	margin-left: 0px;
	margin-top: 0px;
	margin-right: 0px;
	margin-bottom: 0px;
}
  
#container {margin:50px auto;padding:0px;width:910px;}
#left-button{ width:90px; height:100%;float:left }
#middle{ width:730px; margin:0px; float: left;}
#right-button{ width:90px; float:left;}
html, body, div, ul, li, dl, dt, dd, h1, h2, h3, h4, h5, h6, form, input, dd,select, button, textarea, iframe, table, th, td { margin: 0; padding: 0; }
img { border: 0 none; displayblock}
span,ul,li,image{ padding:0px; margin:0px; list-style:none;}
#top-title{ width:714px; margin-bottom:10px; float:left; padding:0px 8px}
#top-title2{ width:320px;  float:right;}
#content{ width: 730px; float:left;float:left;word-wrap: break-word;}
#content-top{ width:730px; background-image:url(images/top_bg.jpg); height:56px; background-repeat:no-repeat; float:left;}
#content-top1{ width:70%; padding:16px 0px 0px 30px; font-family:Arial, Helvetica, sans-serif; font-size:24px; color:#636363;float:left;}
#content-top2{ width:60px; float:right; padding:25px 30px 0px 0px;font-family:Arial, Helvetica, sans-serif; font-size:11px; color:#2f2f2f; font-weight:bold; text-align:right}
#content-middle{ width:730px; background-image:url(images/middle_bg.jpg); background-repeat:repeat-y;float:left;}
#content-middle1{ width:475px; padding:25px 0px 25px 35px; font-family:Arial, Helvetica, sans-serif; font-size:12px; color:#636363; line-height:150%;float:left;}
#content-middle2{  padding:7px; float:right; margin:25px 35px 0px 0px; border:solid 1px #c8c8c9; background-color:#FFF; text-align:center}
#content-bottom{ width:730px; background-image:url(images/bottom_bg.jpg); background-repeat:no-repeat; height:58px;float:left;}
#content-bottom2{ width:110px; height:42px; margin:0px auto; padding-top:4px;}
#input{ width:58px; height:20px; background-color:#FFF; border:solid 1px #dfdfe0; color:#636363; text-align:center; font-size:11px;}
.font1{ font-family:Arial, Helvetica, sans-serif; font-size:10px; color:#2f2f2f;}
.font1 a:link{
	text-decoration: none;
    color:#2f2f2f;

}
.font1 a:visited{
	text-decoration: none;
    color:#2f2f2f;
	
}
.font1 a:hover{
	color: #2f2f2f;
	text-decoration: underline;

}



.text-container {
padding: 20px;
color: #333;
margin-top: 24px;
line-height: 22px;
border-top-left-radius: 3px;
border-top-right-radius: 3px;
}

.use-as-video {
float: right;
padding-left: 20px;
}
.cc-float-fix:before, .cc-float-fix:after {
content: "";
line-height: 0;
height: 0;
width: 0;
display: block;
visibility: hidden;
}
.use-as-video-img {
cursor: pointer;
position: relative;
width: 320px;
height: 180px;
}
.use-as-video-title {
font-size: 15px;
font-weight: bold;
color: #000000;
padding-bottom: 5px;
}
-->
</style>
<script type="text/javascript">
<!--
function MM_preloadImages() { //v3.0
  var d=document; if(d.images){ if(!d.MM_p) d.MM_p=new Array();
    var i,j=d.MM_p.length,a=MM_preloadImages.arguments; for(i=0; i<a.length; i++)
    if (a[i].indexOf("#")!=0){ d.MM_p[j]=new Image; d.MM_p[j++].src=a[i];}}
}

function MM_swapImgRestore() { //v3.0
  var i,x,a=document.MM_sr; for(i=0;a&&i<a.length&&(x=a[i])&&x.oSrc;i++) x.src=x.oSrc;
}

function MM_findObj(n, d) { //v4.01
  var p,i,x;  if(!d) d=document; if((p=n.indexOf("?"))>0&&parent.frames.length) {
    d=parent.frames[n.substring(p+1)].document; n=n.substring(0,p);}
  if(!(x=d[n])&&d.all) x=d.all[n]; for (i=0;!x&&i<d.forms.length;i++) x=d.forms[i][n];
  for(i=0;!x&&d.layers&&i<d.layers.length;i++) x=MM_findObj(n,d.layers[i].document);
  if(!x && d.getElementById) x=d.getElementById(n); return x;
}

function MM_swapImage() { //v3.0
  var i,j=0,x,a=MM_swapImage.arguments; document.MM_sr=new Array; for(i=0;i<(a.length-2);i+=3)
   if ((x=MM_findObj(a[i]))!=null){document.MM_sr[j++]=x; if(!x.oSrc) x.oSrc=x.src; x.src=a[i+2];}
}
//-->
</script>
</head>

<body onload="MM_preloadImages('images/next2.png','images/previous2.png')">
<div id="container">
<div id="left-button">
  <p>&nbsp;</p>
  <p>&nbsp;</p>
  <p>&nbsp;</p>
  <p>&nbsp;</p>
  <p>&nbsp;</p>
  <p><a href="page139.html"><img src="images/previous_big.png" width="90" height="80" /></a></p>
</div>
<div id="middle">
  <div id="top-title"><span class="font1">Basic HTML Version</span>
    <div id="top-title2">
      <table width="100%" border="0" align="right" cellpadding="0" cellspacing="0">
        <tr>
        	<td width="66%"> </td>
          <td width="35%" style="display:none" align="right"><img src="images/content.png" width="19" height="16" /></td>
          <td width="31%" style="display:none" class="font1"><a href="">Table of Contents</a></td>
          <td width="6%"><img src="images/view.png" width="19" height="16" /></td>
          <td width="28%" class="font1"><a href="../../Thesis_JEVI.html#p=140">View Full Version</a></td>
        </tr>
      </table>
    </div>
  </div>
  <div id="content">
     <div id="content-top">
        <div id="content-top1">Page 140 - Thesis_JEVI</div>
        <div id="content-top2">P. 140</div>
     </div>
     <div id="content-middle">
     	
     	  <div class="text-container">
    <div class="use-as-video cc-float-fix">
      <a style="text-decoration: none;" href="../../Thesis_JEVI.html#p=140"><img src="../thumb/140.jpg" /></a>
    </div>

    <p class="use-as-video-title"></p>
    <pre style="white-space:pre-line"><code>CHAPTER 5



            quencing. The GIAB SV truth set was generated by intersecting these three GIAB SV sets
            resulting in a set of 1,515 germline SVs. We used ⅔ of the GIAB truth set as a training set
            and ⅓ as a test set. We established a precision-recall curve from 100 bootstrapping runs
            (Suppl. Figure 4), where the training data were split into 90%-10% train-test subsets.
            Based on the precision-recall curve, we defined an operating point of 96% precision and
            99.5% recall (Suppl. Figure 4). The final model was then re-trained on the whole train-
            ing set and tested on the ⅓ test set. The performance on the test set was 95.1% precision
            and 99.6% recall, representing an accuracy of 97.2% (Suppl. Figure 4). SV candidates are
            classified as “true” or “false” based on this RF model.
            We set up two databases of SV calls: (i) SharcDB: containing raw NanoSV calls from
            nanopore sequencing data of 14 samples, 11 of which belong to this study (COLO829,
            HGS-3, Ova1, Ova2, Ova3, Ova4, Pros1, Pros2, Pros4, Pros5 and Pros6; and three more
            for which we had SV calls from high coverage nanopore data: COLO829BL (lympho-
            blastoid cell line, 50x sequencing depth), VCAP (prostate cancer cell line ) and the Ge-
                                                                       371
     5      nome in a Bottle SV calls (GIAB,  , . For tests performed with the samples included in                                                                                          5
                                        183
            this study, the specific sample was excluded from blacklisting with SharcDB; (ii) RefDB:
            containing germline calls obtained from WGS short-read data of 59 controls: 19 blood
                                              278
            controls from patients with ovarian cancer , where germline SVs were called with Man-
                     350
            ta (v. 1.0.3)  with default parameters and 40 healthy individuals (biological parents of
            individuals with congenital abnormalities)  where germline SVs were called with Man-
                                               372
                       350
            ta (v. 0.29.5)  with default parameters.SV calls from tumor samples are overlapped
            with those two databases by using VCF-explorer (https://github.com/UMCUGenetics/
            vcf-explorer). Only samples classified as “true” by the RF model and that do not overlap
            with any sample in the databases qualify for primer design. Primer design for filtered
            SV calls is automatized by using Primer3 (v. 1.1.4)  with a product size range of 30-230
                                                     349
            bp. SVs with a successful primer design are ranked based on SV length and the 20 larg-
            est are selected for PCR validation. Insertions are filtered out early in the pipeline since
            the inserted sequence cannot be accurately inferred from the low coverage nanopore se-
            quencing data. Inter-chromosomal translocations are not present in the top20 ranked
            SVs because the final ranking is based on SV size and this cannot be determined for
            inter-chromosomal SVs. However they are available in the final VCF file and primers
            are designed by default, so they can be manually selected for PCR validation and assay
            development.

            Breakpoint PCR: To validate SVs, breakpoint PCR with AmpliTaqGold (Applied Bio-
            systems) was performed according to the manufacturer’s protocol. 10 ng primary tumor
            DNA (somatic) and 10 ng lymphocyte DNA (germline) per primer-pair were used as
            input. PCR products were loaded and visualized on a 2% agarose gel.





         132</code></pre>
     	</div>
    
     </div>
     <div id="content-bottom">
       <div id="content-bottom2">
         <table width="110" border="0" cellspacing="0" cellpadding="0">
           <tr>
             <td width="22%" height="44"><a href="page139.html"><img src="images/previous.png" width="26" height="19" id="Image2" onmouseover="MM_swapImage('Image2','','images/previous2.png',1)" onmouseout="MM_swapImgRestore()" /></a></td>
             <td width="43%">
             	&nbsp;&nbsp;&nbsp;<a href="page135.html">135</a>&nbsp;&nbsp;&nbsp;<a href="page136.html">136</a>&nbsp;&nbsp;&nbsp;<a href="page137.html">137</a>&nbsp;&nbsp;&nbsp;<a href="page138.html">138</a>&nbsp;&nbsp;&nbsp;<a href="page139.html">139</a>&nbsp;&nbsp;&nbsp;<a href="page140.html">140</a>&nbsp;&nbsp;&nbsp;<a href="page141.html">141</a>&nbsp;&nbsp;&nbsp;<a href="page142.html">142</a>&nbsp;&nbsp;&nbsp;<a href="page143.html">143</a>&nbsp;&nbsp;&nbsp;<a href="page144.html">144</a>&nbsp;&nbsp;&nbsp;<a href="page145.html">145</a>
             </td>
             <td width="35%"><a href="page141.html"><img src="images/next.png" width="26" height="19" id="Image1" onmouseover="MM_swapImage('Image1','','images/next2.png',1)" onmouseout="MM_swapImgRestore()" /></a></td>
           </tr>
         </table>
       </div>
     </div>
  </div>
</div>
<div id="right-button">
  <p>&nbsp;</p>
  <p>&nbsp;</p>
  <p>&nbsp;</p>
  <p>&nbsp;</p>
  <p>&nbsp;</p>
  <p><a href="page141.html"><img src="images/next_big.png" width="90" height="80" /></a></p>
</div>


</div>

</body>
</html>
